compiler/vm/vmops.nim

```diff
@@ -353,6 +354,14 @@ iterator basicOps*(): Override =
     of 2: setResult(a, round(getFloat(a, 0), getInt(a, 1).int))
     else: doAssert false, $n
 
+  override "stdlib.parseutils.parseBiggestFloat", proc(a: VmArgs) {.nimcall.} =
+    var num: BiggestFloat
+    copyMem(num.addr, pointer(a.getHandle(1).p), sizeof(BiggestFloat))
+    let parsed = a.getString(0).parseBiggestFloat(num, int a.getInt(2))
+    a.setFloat(0, num)
+    a.setResult(parsed)
+
```

Comment:
```suggestion
    var num: BiggestFloat
    copyMem(num.addr, a.getHandle(1).rawPointer, sizeof(BiggestFloat))
    let parsed = a.getString(0).parseBiggestFloat(num, int a.getInt(2))
    copyMem(a.getHandle(1).rawPointer, num.addr, sizeof(BiggestFloat))
    setResult(a, parsed)
```

It's almost correct, but instead of using `setFloat` (which just turns the argument register into a float register with the given value), you need to copy the float value to the location.

For making sure the handle has the correct type, you'd use `a.getHandle(1).typ.kind == akFloat and a.getHandle(1).typ.sizeInBytes == sizeof(BiggestFloat)`, but the general assumption in `vmops` - for better or worse - is that the inputs are all properly typed, so I'd personally leave it out.